lesson01/assignment/index2.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style2.css">
+    <title>Assignment Lesson 1 | Array Methods</title>
+</head>
+<body>
+    <h1>Array Methods in the Browser</h1>
+    <form id="first-form">
+        <label>Type the first thing that come accross to your mind:</label>
+            <p>"Remember your word must contain at least 5 characters"</p>
+        <input type="text" class="entry">
+        <input type="submit" value="Click here! And see what happens">
+            <p>This is what you wrote<span></span></p>
+        <button id="slc1">Click</button>
+            <p id="slc"></p>
+    </form>
+
+    <form id="second-form">

```

Comment:
Remember that classes and ids should be semantic as much as possible. Some of the ones on this page definitely are (like "#show-season" on line 24), but these form ids could be better. An id that indicates the purpose or functionality would be better, like "#season-form" here, "#month-form" below, etc.